deploy/lambda/whitelist.py

```diff
@@ -0,0 +1,80 @@
+import json
+import boto3
+import botocore
+from botocore.vendored import requests
+
+# Config global var before deployment
+REMOVE_PORTS = []
+WHITELIST_SERVICE = 'https://my.service.org/whitelist'
```

Comment:
document what these global vars are designated for

Reply:
its documented in the main lambda below